README.md

```diff
@@ -9,6 +9,57 @@ Terraform Conjur provider
 
 ---
 
+## Installation
+
+### Binaries (Recommended)
+The recommended way to install `terraform-provider-conjur` is to use the binary distributions from this project's
+[GitHub Releases page](https://github.com/cyberark/terraform-provider-conjur/releases).
+The packages are available for Linux, macOS and Windows.
+
+Download and uncompress the latest release for your OS. This example uses the linux binary.
+
+```sh-session
+$ wget https://github.com/cyberark/terraform-provider-conjur/releases/download/v0.1.0/terraform-provider-conjur-linux-amd64.tar.gz
+$ tar -xvf terraform-provider-conjur*.tar.gz
+```
+
+Now copy the binary to the Terraform's plugins folder, if is your first plugin maybe isn't present.
```

Comment:
This could be "Now copy the binary to the Terraform's plugins folder. (If this is your first plugin, you'll need to create the folder first.)"